docker-images/bundle/Makefile

```diff
@@ -0,0 +1,70 @@
+PROJECT_NAME=bundle
+
+CRD_DIR := ../../install/cluster-operator
+DEPLOYMENT_FILE := $(CRD_DIR)/060-Deployment-strimzi-cluster-operator.yaml
+MANIFESTS := ./manifests
```

Comment:
`manifests` seems very generic. Maybe `olm-manifests` would be more corresponding? I'm also not sure what do you need it for. Why not just create some `tmp` file here to keep the files? And maybe the CSV file can live either in `install` or here as well.